scripts/test-wasmer_sys_prod.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+export RUST_BACKTRACE=full
+export WASMER_BACKTRACE=1
+
+# static tests
+cargo fmt
```

Comment:
Oh yes it should be

and you'd think, but this project isn't one workspace. It's actually 2 workspaces and one standalone project. There's the root workspace that contains the host and common crates. Then the test workspace and a standalone guest crate. It drives me potty working on this repo because you can't easily verify changes. I don't think it was done for a reason except maybe to work around historical Cargo issues. I don't think there's an issue open for this, but I'll open one now

Reply:
and not just this actually, we weren't denying warnings for Clippy... so we're just printing them in CI. That's helpful :D Sorted that out and fixed the lint issues that of course existed when we don't enforce them not going in